web/packages/teleterm/src/ui/DocumentCluster/useUserPreferences.ts

```diff
@@ -218,33 +205,3 @@ export function useUserPreferences(clusterUri: ClusterUri): {
     ),
   };
 }
-
-// TODO(gzdunek): DELETE IN 16.0.0.
-// Support for UnifiedTabPreference has been added in 14.1 and for
-// UnifiedViewModePreference in 14.1.5.
-// We have to support these values being undefined/unset in Connect v15.
-function mergeWithDefaultUnifiedResourcePreferences(
```

Comment:
We may not need it anymore, but perhaps we could keep a version anyway? I'm ok with not deleting this for the purpose of catching defaults in tests and what not. Would like your input @gzdunek 🙏

Reply:
This can be removed.